src/pages/docs/quickstart.md

```diff
@@ -55,6 +55,10 @@ The CLI content-addresses your files, packs them into 1 or more CAR files, and u
 
 https://w3s.link/ipfs/bafybeib4ht2a53pttgipw6mgckqqhmgkifnmh2glzsju2c6ait5ibnkow4
 
+By default, `w3` will wrap uploaded files into a folder, so that filename is preserved. It can then be accessed directly by adding the name in the URL path:
```

Comment:
```suggestion
By default, `w3` will wrap files in a folder, so that their filename is preserved. They can then be accessed directly by adding their name in the URL path:
```